source/concepts/applications.rst

```diff
@@ -4,99 +4,91 @@ Applications
+embody themselves with "avatars" and manipulate objects (e.g the simulation box
+in MD).
 
-Each user that is visible to the others have :ref:`an avatar
-<avatar-description>`, the characteristic of which it shares using the
-:ref:`state service <state-service>`. An :ref:`origin
-<user-origin-description>` can optionally be associated to a user. That origin
-is a suggestion from the server to place a users relative to each other.
-Finally, a user can optionally describe its :ref:`play space
-<play-space-description>`, being the area around the user in which it is deemed
-safe to move.
+Clients can broadcast avatar representations of themselves, information about
+their range of motion within shared space (useful for VR), and receive a server
+suggestion about how to position themselves relatives to other clients e.g.
+:ref:`radially oriented <radial-orient>` for distributed setups.
 
-Avatars, user origins, and play spaces are linked to a user with a player ID.
-This ID is an arbitrary string chosen by the user's client. Commonly, clients
+Each user chooses a unique "player ID" for themselves to distinguish the
+ownership and applicability of the user information shared. Commonly, clients
 use a `UUID4
 <https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)>`_
 to reduce the risk of collision between IDs produced by different clients.
-However, any non-empty string is a valid player ID.
-
-.. warning::
-
-   An empty string is an invalid player ID only by convention. At this time, no
-   server will prevent the use of one. However, an empty player ID is more
-   likely to enter in collision between clients which could cause a client's
-   avatar to be overwritten by another client.
-
-A multiplayer application can optionally implement the :ref:`radial orient
-<radial-orient>` method to place the avatars in a circle around a point.
 
 .. _multiplayer-coordinate-systems:
 
 Coordinate systems
 ~~~~~~~~~~~~~~~~~~
 
-The multiplayer application distinguishes between two coordinate spaces:
+The multi-user application distinguishes between two coordinate spaces:
 
-* the game space is the coordinate space of the client. It is fully the
-  responsibility of the client and the server is not aware of any of its
-  details.
-* the server space is the space in which all coordinate through the server are
-  expressed. It is a left-handed, Y-up, coordinate space, with lengths
-  expressed in meters. The origin of the height is on the floor. It is modeled
-  after Unity's coordinate system.
+* The **server space** is the coordinate system of the shared virtual space.
+  The 3d poses of avatars, the simulation box, and any other objects are
+  exchanged in this coordinate space. The characteristics are chosen to match
+  Unity's VR: left-handed, with Y-up, lengths expressed in meters, and the origin
+  at floor level.
+* A **client space**, the local coordinate space of a client, may differ from
```

Comment:
```suggestion
* A **client space** is the local coordinate space of a given client, and which may differ from
```

Or something like this. Just to make it written in a consistent way with the previous bullet point.